File: PHP/Aksesoris.php
<?php
require_once "PetShop.php";

// kelas turunan dari PetShop: Aksesoris
class Aksesoris extends PetShop {
    protected $jenis;
    protected $bahan;
    protected $warna;

    // konstruktor
    public function __construct($id, $nama_produk, $harga, $stok, $foto_produk, $jenis, $bahan, $warna) {
        parent::__construct($id, $nama_produk, $harga, $stok, $foto_produk);
        $this->jenis = $jenis;
        $this->bahan = $bahan;
        $this->warna = $warna;
    }

    // getter
    public function getJenis() { return $this->jenis; }
    public function getBahan() { return $this->bahan; }
    public function getWarna() { return $this->warna; }
}
?>

File: Python/Aksesoris.py
from PetShop import PetShop

# kelas turunan dari PetShop: Aksesoris
class Aksesoris(PetShop):
    def __init__(self, id, nama_produk, harga, stok, jenis, bahan, warna):
        super().__init__(id, nama_produk, harga, stok)
        self.jenis = jenis
        self.bahan = bahan
        self.warna = warna

    # getter
    def get_jenis(self):
        return self.jenis

    def get_bahan(self):
        return self.bahan

    def get_warna(self):
        return self.warna